frontend/src/lib/components/theme_selector.svelte

```diff
@@ -50,17 +50,17 @@
         /></svg
       >
     {/if}
-  </div>
+  </button>
   <ul
-    tabindex="0"
+    tabindex="-1"
     class="p-2 shadow menu dropdown-content bg-base-100 rounded-box w-52"
     style:visibility={isDropdownOpen ? "visible" : "hidden"}
   >
     {#each THEMES as theme}
       <li
         data-set-theme={theme.value}
         data-act-class="ACTIVECLASS"
-        on:click={() => ($chosenTheme = theme.value)}
+        on:keypress={() => ($chosenTheme = theme.value)}
```

Comment:
Having an event on keypress seems a little redundant, since the selector is not navigational with keyboard at all <img src="https://media0.giphy.com/media/1O43UNtta63S5KLLcK/giphy.gif"/>